python/docs/source/getting_started/install.rst

```diff
@@ -159,6 +159,9 @@ Package       Minimum supported version Note
 `NumPy`       1.7                       Required for ML 
 `pyarrow`     1.0.0                     Optional for SQL
 `Py4J`        0.10.9.2                  Required
+`pandas`      0.23.2                    Required for Pandas-on-spark
+`pyarrow`     0.10                      Required for Pandas-on-spark
+`Numpy`       1.14(<1.20.0)             Required for Pandas-on-spark
```

Comment:
`pandas-on-Spark`?

Reply:
or "PySpark APIs on Spark" which will be the official name. Either one should be fine because `ML` is used over `MLlib DataFraem-based API` as its official name (see http://spark.apache.org/docs/latest/ml-guide.html#announcement-dataframe-based-api-is-primary-api)

Reply:
`pandas-on-Spark` is used for now to keep consistent with `ML` for example.